app/test_main.py

```diff
@@ -1 +1,99 @@
-# write your code here
+import pytest
+from unittest import mock
+from requests.exceptions import (
+    MissingSchema,
+    ConnectionError,
+    InvalidURL
+)
+
+from app.main import can_access_google_page
+
+
+@mock.patch("app.main.valid_google_url")
+def test_valid_google_url_works_in_can_access_google_page(
```

Comment:
The test naming  could be improved  test_has_internet_connection_is_called_in_can_access_google_page